docs/book/features/single_access_filesystem_volume.md

```diff
@@ -31,17 +31,17 @@ within a cluster, each with a custom set of parameters.
 The details for provisioning volume using topology and use of `WaitForFirstConsumer` volumeBinding mode with dynamic
 volume provisioning is described [here](volume_topology.md)
 
-**NOTE:** The support for Volume topology is present only in Vanilla Kubernetes Block Volume driver today.
+**NOTE:** The support for Volume topology is present only in Vanilla Kubernetes for single-access(RWO) file system based Volume.
```

Comment:
can you add a space between `access(RWO)`

Reply:
done.